gbpservice/neutron/nsf/modules/configurator.py

```diff
@@ -0,0 +1,119 @@
+import oslo_messaging as messaging
+from oslo_log import log
+from oslo_config import cfg
+from gbpservice.neutron.nsf.core.main import Event
+from gbpservice.neutron.nsf.core.main import RpcAgent
+from gbpservice.neutron.nsf.configurator.lib import fw_constants
+from gbpservice.neutron.nsf.configurator.lib import vpn_constants
+from gbpservice.neutron.nsf.configurator.lib import lb_constants
+
+from gbpservice.neutron.nsf.configurator.agents.firewall import \
+                            FWaasRpcManager, FWaasEventHandler
+from gbpservice.neutron.nsf.configurator.agents.vpn import \
+                            VPNaasRpcManager, VPNaasEventHandler
+from gbpservice.neutron.nsf.configurator.agents.loadbalancer import \
+                            LBaasRpcManager, LBaasEventHandler
+
+LOG = log.getLogger(__name__)
+
+
+def rpc_init(sc, conf):
+    # FWaaS agent
+    fw_agent_state = {
+            'start_flag': True,
+            'binary': fw_constants.OC_FW_AGENT_BINARY,
+            'host': cfg.CONF.host,
+            'topic': fw_constants.FIREWALL_RPC_TOPIC,
+            'plugin_topic': fw_constants.OC_FW_PLUGIN_TOPIC,
+            'agent_type': fw_constants.OC_AGENT_TYPE,
+            'configurations': {
+                'driver': fw_constants.OC_FIREWALL_DRIVER
+            },
+            'report_interval': cfg.CONF.ocfwaas.oc_report_interval,
+            'description': 'oc firewall agent '
+        }
+
+    fw_rpc_receiver = FWaasRpcManager(conf, sc)
+    fw_agent = RpcAgent(sc,
+                        fw_constants.FIREWALL_RPC_TOPIC,
+                        fw_rpc_receiver,
+                        fw_agent_state)
+
+    # VPNaaS agent
+    vpn_rpc_receiver = VPNaasRpcManager(conf, sc)
+
+    vpn_agent = RpcAgent(sc,
+                         vpn_constants.VPN_RPC_TOPIC,
+                         vpn_rpc_receiver)
+
+    # LBaaS agent
+    lb_agent_state = {
+        'binary': 'nsf-lb-module',
+        'host': conf.host,
+        'topic': lb_constants.LBAAS_AGENT_RPC_TOPIC,
+        'report_interval': 10,
+        'plugin_topic': lb_constants.LBAAS_PLUGIN_RPC_TOPIC,
+        'configurations': {'device_drivers': 'haproxy_on_vm'},
```

Comment:
For now passing device_drivers this way. Will have to modify when multivendor support comes